resources/js/components/ui/calendar/ColumnsCalendar.tsx

```diff
@@ -0,0 +1,178 @@
+import {currentDate, currentTime, cx, htmlAttributes} from "components/utils";
+import {DateTime} from "luxon";
+import {
+  For,
+  Index,
+  JSX,
+  Show,
+  VoidComponent,
+  createContext,
+  createEffect,
+  createMemo,
+  mergeProps,
+  on,
+  splitProps,
+  useContext,
+} from "solid-js";
+import s from "./ColumnsCalendar.module.scss";
+import {MAX_DAY_MINUTE, formatDayMinuteHM, getDayMinute} from "./util";
+
+interface GlobalParameters {
+  readonly pixelsPerHour?: number;
+  readonly gridCellMinutes?: number;
+}
+
+interface Props extends GlobalParameters, htmlAttributes.div {
+  readonly columns: readonly CalendarColumn[];
+  readonly scrollToDayMinute?: number;
+
+  readonly onWheelWithAlt?: (e: WheelEvent) => void;
```

Comment:
Użyłem alta, bo czytałem kod, ale zastanawiam się, czy przeskalowanie kalendarza nie byłoby instynktowniejsze z ctrl.

Reply:
Chciałem tak zrobić, ale scrollowanie z ctrl robi zooma przeglądarki, i jakoś nie chciałem tego blokować, bo komuś to też może się przydać. Więc chyba zostawię na razie alt, ale dopisałem do mojej listy sprytnych tricków z zerowym discoverability :) Jak kiedyś zrobimy system samouczków to się to tam wrzuci.